cluster/engine.go

```diff
@@ -921,8 +921,6 @@ func (e *Engine) CreateContainer(config *ContainerConfig, name string, pullImage
 	// Register the container immediately while waiting for a state refresh.
 	// Force a state refresh to pick up the newly created container.
 	e.refreshContainer(createResp.ID, true)
-	e.RefreshVolumes()
```

Comment:
@xianlubird Please keep e.RefreshVolumes() which is required to update engine volume manifest. 

And remove `e.RefreshNetworks()`.